shared/fixes/bash/dconf_gnome_disable_wifi_create.sh

```diff
@@ -0,0 +1,7 @@
+# platform = Red Hat Enterprise Linux 7
```

Comment:
if it's rhel7 only don't put it in shared, if it's universal don't make it rhel7 only

Reply:
The majority of rules in shared are rhel7 only.

Reply:
@mpreisler added to Fedora as well.

Reply:
> The majority of rules in shared are rhel7 only.

yes and that sucks, let's not make it even worse :D